src/main/java/com/interview/solutions/algorithm/FindMaxRange.java

```diff
@@ -1,4 +1,4 @@
-package com.jthomas.algorithm;
+package com.interview.solutions.algorithm;
```

Comment:
Package names need to be changed to something else

Reply:
These are all redundant comments pointing to one question which is "change the name of the repo".

Reply:
Yeah